### PR TITLE
Fix user feedback

### PR DESCRIPTION
**Pull request by kirjs**
Also normalize all headers and make them smaller

### CODE REVIEW

File: ng2ts/tests/templatePageSetupTest.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ describe('Blabla', () => {
     chai.expect(header2.innerHTML, `Use the curly braces to put component title property in the header`).contains('SomethingElse');
   });
 
-  it(`app.html: Add an <input> with a 'placeholder' set to 'video' (hint: 'placeholder' is just an attribute of an input tag) `, () => {
+  it(`app.html: Add an <input> with a 'placeholder' atribute set to 'video'`, () => {
```
Comment (prestonvanloon) [inline ng2ts/tests/templatePageSetupTest.ts]: `Attribute`

File: src/app/demo/feedback-page/feedback-page.component.ts [hunk 2]
```diff
@@ -87,39 +88,23 @@ export class FeedbackPageComponent implements OnInit {
...
     this.ghService.createIssue({
-      title: 'NO FIX: ' + message.comment.substring(0, 150),
+      title: reason + ' ' + message.comment.substring(0, 150),
       body: this.generateIssueBody(message),
     }, this.githubAuth.credential.accessToken).subscribe(response => {
       if (response.ok) {
         const responseData = response.json();
-        this.database.object(`feedback/${message.$key}`).update({ url: responseData.html_url });
-        this.ghService.closeIssue({ state: 'closed' }, responseData.number, this.githubAuth.credential.accessToken)
+        console.log(responseData.html_url);
```
Comment (prestonvanloon) [inline src/app/demo/feedback-page/feedback-page.component.ts]: Delete

File: src/app/demo/typescript/typescript.component.html [hunk 3]
```diff
@@ -29,8 +25,8 @@
       <div>
 
         <ul>
-          <li><b>TypeScript</b> a language transpiled to JavaScript. It's based on (<b>ES5</b>), the standard, current
-            version of JavaScript supported by modern browsers.
+          <li><b>TypeScript</b> a language transpiled to JavaScript. It's extends <b>ES5</b> standard (current
```
Comment (prestonvanloon) [inline src/app/demo/typescript/typescript.component.html]: `It`

File: src/app/demo/typescript/typescript.component.html [hunk 4]
```diff
@@ -136,33 +133,33 @@ <h1 data-step="1" data-intro="Welcome to the first coding exercise!">TypeSystem<
 
   <!--  CLASSES  -->
   <slides-slide id="classes" milestone="TypeScript">
-    <h1>Classes</h1>
-    <h2>TypeScript has classes, and Angular makes heavy use of them</h2>
-    <h2>They work similar to classes in other languages, and serve for grouping methods and properties belonging to
+    <h1>TypeScript / Classes</h1>
+    <h2>TypeScript has classes, and Angular uses them heavily</h2>
+    <h2>They are similar to classes in other languages. Used to group methods and properties belonging to
       one object together</h2>
```
Comment (prestonvanloon) [inline src/app/demo/typescript/typescript.component.html]: Missing punctation.

File: src/app/presentation/presentation/presentation.component.ts [hunk 5]
```diff
@@ -75,7 +75,7 @@ export class PresentationComponent implements OnInit {
     if (this.activeSlideIndex === 0) {
       const key = `been-here-mileston-start-${path}`;
       const beenHere = localStorage.getItem(key);
-      const time = Math.floor(Date.now() / 1000);
+      const time = Math.floor(+Date.now());
```
Comment (prestonvanloon) [inline src/app/presentation/presentation/presentation.component.ts]: Is this necessary? Why floor `Date.now()`?

File: src/app/presentation/presentation/presentation.component.ts [hunk 6]
```diff
@@ -88,10 +88,11 @@ export class PresentationComponent implements OnInit {
 
       if (!beenHere) {
         const startTime = parseInt(localStorage.getItem(`been-here-mileston-start-${path}`), 10);
-        const time = Math.floor(Date.now() / 1000) - startTime;
+        const time = Math.floor(+Date.now()) - startTime;
         localStorage.setItem(key, 'yes');
         ga('send', 'event', 'milestone', path, 'end');
-        ga('send', 'timing', 'milestone', 'complete', time);
+        ga('send', 'timing', 'milestone', 'complete', time, path);
+        console.log('milestone-complete', time);
```
Comment (prestonvanloon) [inline src/app/presentation/presentation/presentation.component.ts]: Delete

File: ng2ts/tests/templatePageSetupTest.ts (hunk 1 above)
Comment (kirjs) [inline ng2ts/tests/templatePageSetupTest.ts]: Fixed in the next commit

File: src/app/demo/feedback-page/feedback-page.component.ts (hunk 2 above)
Comment (kirjs) [inline src/app/demo/feedback-page/feedback-page.component.ts]: That's actually intentional, added a comment in the next PR

File: src/app/presentation/presentation/presentation.component.ts (hunk 5 above)
Comment (kirjs) [inline src/app/presentation/presentation/presentation.component.ts]: Good catch, got rid of it.

File: src/app/presentation/presentation/presentation.component.ts (hunk 6 above)
Comment (kirjs) [inline src/app/presentation/presentation/presentation.component.ts]: fixed in next PR

**Pull request merged**